src/main/java/seedu/duke/command/AddNoteCommand.java

```diff
@@ -48,17 +52,31 @@ public String execute() {
         }
 
         // Get Content
+        try {
+            if (storageManager.noteExists(note)) {
+                content = storageManager.getNoteContent(note);
+            }
+        } catch (SystemException exception) {
+            return Formatter.formatString(exception.getMessage());
+        }
+
         if (content.isEmpty()) {
             content = inputContent();
         }
-
         // Edit the note
         note.setContent(content);
 
         // Rebind the tags if there are duplicated tags
         tagManager.rebindTags(note);
         notebook.addNote(note);
 
+        //Save the notes in storage
+        try {
+            StorageManager.saveNote(note);
```

Comment:
Would there be any difference if saveNote is not a static function? Since all command classes contains a storageManager object, all are able to access this function.

Reply:
It creates an error

Reply:
Would you be able to create a StorageManager storageManager variable and access storageManager.saveNote(note)?

Reply:
What error? I tried and it works.